string text
bool is_final
float32 processing_time_s
int64 total_samples_count
